.travis.yml

```diff
@@ -64,15 +64,26 @@ matrix:
       script: skip
     - stage: deploy
       if: branch = master AND type != pull_request AND commit_message =~ /^Bump version/
-      script:
-        - docker load -i $HOME/docker/images.tar || true
-        - docker image list
-        - docker build --cache-from ${GOOGLE_DOCKER_IMAGE} --build-arg COMMIT=$TRAVIS_COMMIT -t ${IMAGE_BRANCH_COMMIT} .
-        - make gcloud-login
-        - IMAGE=${IMAGE_BRANCH_COMMIT} make docker-push
-        - IMAGE=${IMAGE_BRANCH_COMMIT} make pubsub-publish
-        - docker tag ${IMAGE_BRANCH_COMMIT} ${GOOGLE_DOCKER_IMAGE}
-        - IMAGE=${GOOGLE_DOCKER_IMAGE} make docker-push
+      script: |
+        docker load -i $HOME/docker/images.tar || true
+        docker image list
+        VERSION_FILE=server/athenian/api/metadata.py
+        CURRENT_VERSION=$(grep __version__ $version_file | cut -d\" -f2)
+        IMAGE=${IMAGE_BRANCH_COMMIT}-${CURRENT_VERSION}
+        docker build --cache-from ${GOOGLE_DOCKER_IMAGE} --build-arg COMMIT=$TRAVIS_COMMIT -t ${IMAGE} .
+        make gcloud-login
+        make docker-push
+        make pubsub-publish
+        docker tag ${IMAGE} ${GOOGLE_DOCKER_IMAGE}
+        IMAGE=${GOOGLE_DOCKER_IMAGE} make docker-push
+    - stage: deploy-production
+      if: branch = production AND type != pull_request AND commit_message =~ /^Bump version/
+      script: |
+        make gcloud-login
+        VERSION_FILE=server/athenian/api/metadata.py
+        CURRENT_VERSION=$(grep __version__ $version_file | cut -d\" -f2)
+        IMAGE=${IMAGE_BRANCH_COMMIT}-${CURRENT_VERSION}
+        make pubsub-publish
```

Comment:
How is it different from the regular `deploy`? I mean, both do `make pubsub-publish`, so will this always be a no-op?

Reply:
@vmarkovtsev do you propose to put them in one job?

Reply:
Done